sources/packages/web/src/components/generic/StatusChipRequestedAssessment.vue

```diff
@@ -1,27 +1,39 @@
 <template>
-  <chip-status :status="chipStatus" :label="status" />
+  <chip-status :status="chipStatus" :label="chipStatusLabel" />
 </template>
 <script lang="ts">
 import { computed, defineComponent, PropType } from "vue";
 import ChipStatus from "@/components/generic/ChipStatus.vue";
 import { useAssessment } from "@/composables";
-import { StudentAppealStatus, ApplicationExceptionStatus } from "@/types";
+import {
+  StudentAppealStatus,
+  ApplicationExceptionStatus,
+  ApplicationOfferingChangeRequestStatus,
+} from "@/types";
 export default defineComponent({
   components: { ChipStatus },
   props: {
     status: {
       type: String as PropType<
-        StudentAppealStatus | ApplicationExceptionStatus
+        | StudentAppealStatus
+        | ApplicationExceptionStatus
+        | ApplicationOfferingChangeRequestStatus
       >,
       required: true,
     },
   },
   setup(props) {
-    const { mapRequestAssessmentChipStatus } = useAssessment();
+    const {
+      mapRequestAssessmentChipStatus,
+      mapRequestAssessmentChipStatusLabel,
```

Comment:
I am not able to follow why did we create an additional mapping for label here.

Reply:
If I am not wrong, only for Students, there was a UI request to change some of them to be displayed differently, so, besides mapping only the statuses to colors it was also needed to have some label "translated" again. I am not defending the idea, just trying to recap what was requested from the UI perspective.
That said, it was only meant to be displayed like that for Students, so I am not sure if the generic component should be the one dealing with it.